src/main/java/com/google/cloud/tools/app/api/deploy/DeployConfiguration.java

```diff
@@ -28,17 +28,17 @@
 
   String getDockerBuild();
 
-  boolean isForce();
```

Comment:
Agreed that there are 2 states. But you can omit that flag altogether from the gcloud command right? Just because it defaults to false now when you omit it doesn't mean that the default couldn't change in the future.